chatbot/src/test/java/chatbot/core/classifier/ClassifierTest.java

```diff
@@ -64,12 +64,12 @@ public void testClassifyForElizaHandler() {
         assertTrue(actualOutput instanceof ElizaHandler);
 	}
 
-	@Test
-	public void testClassifyForSessaHandler() {
-		
-		IncomingRequest input = createInitialRequest("wq");
-		Object actualOutput= classifyInput(input);
-		log.info("Executing SessaHandler Test");
-        assertTrue(actualOutput instanceof SessaHandler);
-	}
+//	@Test
+//	public void testClassifyForSessaHandler() {
+//		
+//		IncomingRequest input = createInitialRequest("wq");
```

Comment:
This is a wrong test case written . Hence it doesn't work. "wq" doesn't make any sense and returning null is the correct response. You don't have to comment the test case but give it proper keywords to search like "Obama wife" or something like that since sessa is still handled by our code and is not removed yet.

Reply:
I think you should know why that test case was written before commenting it wrong. This test case wasn't written wrong and this was not the reason for the test failure. Null wasn't returned because of wq. Sessa had an appropriate respone for such queries. Null was also returned in queries where the test has to pass. Null was returned in all cases by classifier because the rivescript resources where not loaded. Hence the issue for null response got resolved only after guru's commit. This test was written to test whether sessa is able to respond to any random request. This is now commented because we are no longer using sessa.

Reply:
@Juzerhk The test case should remain because sessa code is still there. When Sessa handler file is removed, we can also remove the test case.  Give it meaningful inputs for now since its not a error test case.  And this UT is not testing Sessa, just whether SessaHandler class object is returned or not.